mp/src/game/server/neo/neo_player.cpp

```diff
@@ -1204,9 +1204,14 @@ bool CNEO_Player::ClientCommand( const CCommand &args )
 			switch (slot)
 			{
 			case 1: // Dismiss
+			case 9: // Do not show again
```

Comment:
Could we have an enumeration instead of magic values + comments, to make this clearer